tools/rust-bindgen

```diff
@@ -16,31 +17,6 @@ create_xc_framework() {
   cp $BUILD_FOLDER/catalyst/$TARGET_NAME $TARGET_XCFRAMEWORK_NAME/ios-arm64_x86_64-maccatalyst
 }
 
-find_android_ndk() {
-  if [[ "$ANDROID_NDK_HOME" != "" ]]; then
-    echo "Use ANDROID_NDK_HOME"
-  elif [[ "$ANDROID_HOME" != "" ]]; then
-    echo "ANDROID_NDK_HOME is not set. Use ANDROID_HOME value instead"
-    ANDROID_NDK_HOME="$ANDROID_HOME/ndk"
-  else
-    echo "WARNING: ANDROID_HOME is not set. Use a default path"
-    ANDROID_NDK_HOME="$HOME/Library/Android/sdk/ndk"
-  fi
-
-  TEST_CLANG="aarch64-linux-android$NDK_API_LEVEL-clang"
-  PATH_TO_CLANG=$(find "$ANDROID_NDK_HOME" -iname $TEST_CLANG -print -quit)
-
-  if [[ "$PATH_TO_CLANG" == "" ]]; then
-    echo "ERROR: cannot find NDK tools within '$ANDROID_NDK_HOME'"
-    return 1
-  fi
-
-  NDK_BIN_PATH=$(dirname "$PATH_TO_CLANG")
-  export NDK_BIN_PATH
-
-  return 0
-}
```

Comment:
this will probably be needed for release repository

Reply:
I moved this function into the `tools/android-sdk` script